templates/blender/index.html

```diff
@@ -0,0 +1,100 @@
+{% extends "blender/_base-blender.html" %}
+
+{% block title %}Blender Support Page{% endblock %}
+{% block meta_description %}SMEs and studios can put the world’s most popular 3D creation software into production with Canonical enterprise-grade support.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1UgPrmv0ambszuLtW9mFjT5KYA7nqaubH3zQg6XNqprw/edit#{% endblock meta_copydoc %}
+
+{% block content %}
+
+<section class="p-strip--suru-bottomed">
+  <div class="row">
+    <div class="col-6">
+      <h1>Blender Support from&nbsp;Canonical</h1>
+      <p class="p-heading--4">SMEs and studios can put the world’s most popular 3D creation software into production with Canonical enterprise-grade support</p>
+      <p class="p-heaing--4">Find out how we can support you</p>
```

Comment:
```suggestion
      <p class="p-heading--4">Find out how we can support you</p>
```

Reply:
This sentence also doesn't make sense without the contact-us button. Maybe you should keep the contact us buttons, use the correct marketo form-ID but just use a generic marketo form template for now until the new one is built?